packages/caver-klay/caver-klay-accounts/src/makeRawTransaction.js

```diff
@@ -177,7 +177,44 @@ function _combineFeePayerRawTransaction(rlpEncoded, sig, transaction, senderSign
   return txType + RLP.encode(rawTx).slice(2)
 }
 
+function extractSignatures(rawTransaction) {
+  let senderSignatures = []
+  let feePayerSignatures = []
+
+  const decoded = _decodeFromRawTransaction(rawTransaction)
+  senderSignatures = senderSignatures.concat(decoded.signatures)
+  if (decoded.feePayerSignatures) {
+    feePayerSignatures = feePayerSignatures.concat(decoded.feePayerSignatures)
+  }
+  return { senderSignatures, feePayerSignatures, decodedTransaction: decoded }
+}
+
+function decodeTxForFeePayer(rawTransaction) {
+  const txType = rawTransaction.slice(0, 4)
+  const decodedValues = RLP.decode(utils.addHexPrefix(rawTransaction.slice(4)))
```

Comment:
Also, don't we need to check if the transaction type is a fee-delegated transaction type?

Reply:
Oh, right.
The check logic for fee delegated type is added :)